stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ChannelUtils.kt

```diff
@@ -57,3 +59,47 @@ public fun Channel.getReadStatuses(userToIgnore: User?): List<Date> {
  * @return True if the channel is distinct.
  */
 public fun Channel.isDistinct(): Boolean = cid.contains("!members")
+
+/**
+ * Checks if the channel is a direct conversation between the current user and some
+ * other user.
+ *
+ * A one-to-one chat is basically a corner case of a distinct channel with only 2 members.
+ *
+ * @param currentUser The currently logged in user.
+ * @return True if the channel is a one-to-one conversation.
+ */
+public fun Channel.isOneToOne(currentUser: User?): Boolean {
+    return isDistinct() &&
+        members.size == 2 &&
+        members.any { it.user.id == currentUser?.id }
+}
+
+/**
+ * Returns a string describing the member status of the channel: either a member count for a group channel
+ * or the last seen text for a direct one-to-one conversation with the current user.
+ *
+ * @param context The context to load string resources.
+ * @param currentUser The currently logged in user.
+ * @return The text that represent the member status of the channel.
+ */
+public fun Channel.getMembersStatusText(context: Context, currentUser: User?): String {
+    val otherMembers = members.filter { it.user.id != currentUser?.id }
+
+    return when {
+        otherMembers.isEmpty() -> ""
+        isOneToOne(currentUser) -> {
```

Comment:
I think we can remove the {} as it's only ever going to be one statement!

Reply:
Doesn't agree here as the readability suffers. Anyway, committed the change.

Reply:
+1 on readability, I'd also have kept the branch, but it's fine either way, let's not get lost in the detail 😃

Reply:
We can totally keep the braces if you feel it's more readable, I felt differently from my PoV, but it's 2 against one 😄